src/data/faq.json

```diff
@@ -200,7 +200,7 @@
                         "anchor": "test_cert_get",
                         "active": true,
                         "textblock": [
-                            "You can request the certificate by registering a test in the Corona-Warn-App (CWA): To do this, tap on 'Register Test' on the start screen of your CWA and then on 'Scan QR Code'. After scanning the QR code for test registration, the window for the COVID test certificate automatically opens. There you can 'request the test certificate' or reject it if you do not want to receive one."
+                            "You can request the certificate by registering a test in the Corona-Warn-App (CWA): To do this, tap on 'Manage Your Tests' on the start screen of your CWA (or on 'Register Test' for previous app versions) and then on 'Scan QR Code'. After scanning the QR code for test registration, the window for the COVID test certificate automatically opens. There you can tap 'Request Test Certificate' or decline it if you do not want to receive one."

```

Comment:
> previous app versions

You should avoid generic statements like this. It's better to name a specific version as a point of reference: "since CWA version XXX" or "until CWA version XXX"

Reply:
@heinezen 
You are right. I have changed it to say until version 2.5.